system-addon/content-src/components/Base/_Base.scss

```diff
@@ -1,8 +1,15 @@
 .outer-wrapper {
   display: flex;
-  flex-grow: 1;
   padding: $section-spacing $base-gutter $base-gutter;
   height: 100%;
+  flex-grow: 1;
+
+  &.fixed-to-top {
+    height: auto;
+    @media (min-width: $break-point-large) and (min-height: 800px) {
+      padding-top: 10vh;
```

Comment:
Where did these come from? The existing outer-wrapper seems to have a top padding of 40px, which is `$section-spacing`

Reply:
I was playing around with some stuff to make it look better for larger screens, but I talked to aaron and he figures `height: auto` is fine so i'll remove it